### DISCUSSION TITLE
How to disable Release but still Publish

### DISCUSSION THREAD

**Discussion by xendren** [goreleaser/goreleaser#3123] [Ideas, Issue Triage, and General discussions]
We must be missing something in the settings.  We are running goreleaser in a hosted Gitlab Runner environment.  We are trying to configure goreleaser to NOT add artifacts to the Gitlab Release, but we still want to publish to our external registry.

If we run with Publishers section setup, the pipeline fails because it is trying to publish to the gitlab.com project registry.
If we add a Release section and turn disable to TRUE, then it appears to completely skip the Publishers section.

How can we publish without adding anything to the Gitlab Release entry?

**Comment by caarlos0** [comment]
disabling only the scm release in the config should work... 

passing `--skip-publish` (or `--snapshot`) skips the whole publishing step though.
